### PR TITLE
Created ConsentStatement and ConsentGrant

### PR DESCRIPTION
**Pull request by jhigman**

### CODE REVIEW

File: app/controllers/admin/consent_statements_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          render :edit
+        end
+      else
+        flash[:error] = 'This consent statement is no longer editable'
+        render :edit
+      end
+    end
+
+    def destroy
+      # TODO only if not referenced yet
```
[the remark this answers is not in the document]
Comment (jhigman) [reply app/controllers/admin/consent_statements_controller.rb]: Done

File: app/models/consent_grant.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class ConsentGrant < ApplicationRecord
```
[the remark this answers is not in the document]
Comment (jhigman) [reply app/models/consent_grant.rb:1]: Done

File: app/views/admin/consent_grants/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<h1>Consent Granted</h1>
+
+<p><%= link_to 'All consents granted', admin_consent_grants_path %></p>
+
+<h2>Consent Details</h2>
+
+<ul>
+  <li>School: <%= link_to @consent_grant.school.name, school_path(@consent_grant.school) %></li>
```
[the remark this answers is not in the document]
Comment (jhigman) [reply app/views/admin/consent_grants/show.html.erb:8]: Done

File: app/controllers/admin/consent_statements_controller.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    def show
+    end
+
+    def new
+    end
+
+    def edit
+    end
+
+    def create
```
[the remark this answers is not in the document]
Comment (jhigman) [reply app/controllers/admin/consent_statements_controller.rb:18]: Think we could have more than one in draft - e.g. where 2 versions are being suggested and compared..?

**Comment by ldodds** [comment]
:+1:

**Pull request merged**